end-to-end-test/local/specs/treatment.screenshot.spec.js

```diff
@@ -30,7 +30,7 @@ describe('treatment feature', () => {
             );
             $('.oncoprint__controls__heatmap_menu textarea').setValue('17-AAG');
             $('div.icon-area div.icon').waitForExist();
-            $('button=Add Treatment Response to Heatmap').click();
+            $('button=Add entity to Heatmap').click();
```

Comment:
is entity a comprehensible word to show to users?

Reply:
yeah agreed - this should prolly show entity type (i.e. treatment response) or something

Reply:
Yeah, I can try to use the existing `generic assay type` at here (e.g. `treatment response` or `mutational signature`). If that's not good enough, we can move forward with this https://github.com/cBioPortal/cbioportal/pull/7132